project/yfnijm/README.md

```diff
@@ -0,0 +1,53 @@
+#  R-Tree Search Engine 
+
+## Basic Information
+R-tree is a girdless data structure base on B-tree to speed up searches in space, *R* is stand for *Rectangle*. The average search time would be O(lg(n)) and n is the object number.
+
+## Problem to solve
+To speed up the operation between target(box, segement , point) and the space, using the C++ code to implement the core function. 
+
+To solve the most of EDA problem which only have a few layer, this project would focus on providing the 2D R-tree related API. And it would be usful to the placement and routing problem especially. 
+
+
+## Perspective users
+The student of EDA related Lab.
+And anyone need to speed up the area search in their works.
+
+## System architecture
+The system base on the main data structure class **R-tree** and users can add or delete new object and dynamically modify the data inside. And users also can query the imformation they what in the stucture at all times.
```

Comment:
It will be good to describe what is the initial design of the data structure of the tree.  It's well documented.